queries/util/__init__.py

```diff
@@ -364,23 +359,18 @@ def iceformat_float(
         return strip_trailing_zeros(res) if strip_zeros else res
 
 
-def icequote(s: str) -> str:
-    """Return string surrounded by Icelandic-style quotation marks."""
-    return "„{0}“".format(s.strip())
-
-
 def gen_answer(a: str) -> AnswerTuple:
     """Convenience function for query modules: response, answer, voice answer"""
     return dict(answer=a), a, a
 
 
-def query_json_api(url: str, headers: Optional[Dict[str, str]] = None) -> JsonResponse:
+def query_json_api(url: str, headers: Optional[Dict[str, str]] = None, *, timeout:int=10) -> JsonResponse:
```

Comment:
Formatting of "timeout:int=10" seems wonky. Did black accept this?

Reply:
whoops forgot to format, fixed